mirror-maker-2/active-passive-ssl/kustomization.yaml

```diff
@@ -5,6 +5,12 @@ kind: Kustomization
 resources:
   - deployment.yaml
 
+components:
+  - ../../shared/opslevel
+
+patches:
+  - path: opslevel.yaml
```

Comment:
I think we should decide on either strategic merge or json patch- there is no reason to add both

Reply:
> I think we should decide on either strategic merge or json patch- this PR proposes two ways of applying very similar patches

I guess the consistency is nice....I do like having the YAML in a separate file and not quoted YAML within YAML, but kustomize doesn't want to play nice, so [3a55cc8](https://github.com/utilitywarehouse/kafka-manifests/pull/57/commits/3a55cc8ff43e66880f52f3b3968edd3bdbf14174)